modules/update-upgrading-cli.adoc

```diff
@@ -131,20 +135,45 @@ When using `oc adm upgrade --help`, there is a listed option for `--force`. This
 +
 [source,terminal]
 ----
-$ oc adm upgrade
+$ oc adm upgrade status
 ----
-ifdef::openshift-origin[]
```

Comment:
What you are doing by deleting this line and the endif [here](https://github.com/openshift/openshift-docs/pull/98609/files#diff-a4543a07ef809868d1b552e88f1c5a5b4dbbc99a3ff5c769ec4ad63cd55d0b4dL147) is making content that was previously included _only_ in OKD docs show up in everything else.